suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -152,6 +152,19 @@ export const selectAccountsByNetworkSymbols = memoizeWithArgs(
     },
 );
 
+export const selectAccountsByNetworkSymbol = memoizeWithArgs(
+    (state: AccountsRootState, networkSymbol: NetworkSymbol | null) => {
+        if (G.isNull(networkSymbol)) return [];
+
+        const accounts = selectAccounts(state);
+
+        return A.filter(accounts, account => account.symbol === networkSymbol);
+    },
+    {
+        size: 16,
```

Comment:
nit: Create constant `MAX_NUMBER_OF_COINS` so it's not just a magic number?

Reply:
Also there already seems to be a selector for this. `selectAccountsByNetworkSymbols`

Reply:
ad 1. Exact networks count added in 45943ed69. 

ad 2. The selector you mentioned accepts array of symbols instead a single network symbol as I implemented. I created a new one because I believe that using this new selector is nicer than wrapping argument to an array as: `selectAccountsByNetworkSymbols(state, [specificNetworkSymbolVariable])`

Reply:
But I don't see anything wrong with wrapping it into an array if it avoids duplication. I'd just delete this one and keep the previous selector

Reply:
I just realized that the selector *selectAccountsByNetworkSymbols* is not used anywhere in the code. So I've removed it in d4a62169f.